src/api/spec/models/workflow/step/branch_package_step_spec.rb

```diff
@@ -223,6 +219,57 @@
         it_behaves_like 'fails with insufficient write permission on target project'
       end
 
+      context 'for a multibuild package' do
```

Comment:
This is mostly a copy-and-paste from the `'successful new PR or MR event'` shared context in this file. Shared contexts are a nightmare to maintain and on top of this, it didn't pass for a muiltibuild package, so it had to be changed.